storage/innobase/buf/buf0dump.cc

```diff
@@ -421,11 +425,11 @@ void buf_load_throttle_if_needed(ulint *last_check_time,
   "cur_activity_count == *last_activity_count" check and calling
   ut_time_ms() that often may turn out to be too expensive. */
 
-  if (elapsed_time < 1000 /* 1 sec (1000 milli secs) */) {
-    os_thread_sleep((1000 - elapsed_time) * 1000 /* micro secs */);
+  if (elapsed_tm < ms1000 /* 1 sec (1000 milli secs) */) {
```

Comment:
Code becomes self-descriptive now:
```suggestion
  if (elapsed_tm < ms1000) {
```